### PR TITLE
Default project thumbnails - PT 2

### PR DESCRIPTION
**Pull request by jsaq007**
## What do these changes do?
Screenshot 2024-02-27 at 12 38 06

- Update thumbnails based on one received for S4L
Screenshot 2024-02-27 at 12 36 37

- Added Thhumbnails for oSparc (Generated some temporary one - ideas welcome)
Screenshot 2024-02-27 at 12 37 28

- Removed thumbnail view from project overview (Still being refined but due soon)
Screenshot 2024-02-27 at 12 48 17

## Dev Checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by elisabettai — APPROVED** [review]
Beautiful! 🤩 
Thanks for the creating some thumbnails for oSPARC as well!

Not super-high priority, but maybe you can already kill this case (or we can discuss it)? https://github.com/ITISFoundation/osparc-issues/issues/1212
In that case, I was suggesting:
1. If the thumbnail is not set, and there's no service thumbnail that can be used, use a default that is specific to the product (e.g. for osparc it can be the first in the `osparcTemplates` list)
2. It was not possible anymore to have the thumbnail field empty. I am not sure if that was intentional.

**Review by bisgaard-itis — APPROVED** [review]
Cool! Looks good. Thanks

**Review by pcrespov — COMMENTED** [review]
:+1:

**Review by sanderegg — APPROVED** [review]
nice!

File: services/static-webserver/client/source/class/osparc/editor/ThumbnailSuggestions.js [hunk 1]
```diff
@@ -73,8 +93,25 @@ qx.Class.define("osparc.editor.ThumbnailSuggestions", {
...
+        case "s4ldesktop":
+          return this.self().defaultTemplates;
+        case "s4ldesktopacad":
+          return this.self().defaultTemplates;
+        case "osparc":
+          return this.self().osparcTemplates;
+        case "s4l":
+        default:
+          return this.self().defaultTemplates;
+      }
```
Comment (ignapas) [inline services/static-webserver/client/source/class/osparc/editor/ThumbnailSuggestions.js:114]:
```suggestion
      switch (osparc.product.Utils.getProductName()) {
        case "osparc":
          return this.self().osparcTemplates;
        default:
          return this.self().defaultTemplates;
      }
```

**Review by ignapas — APPROVED** [review]

**Review by GitHK — COMMENTED** [review]
👍

**Review by matusdrobuliak66 — APPROVED** [review]

**Pull request merged**